doc/src/index.md

```diff
@@ -115,14 +115,29 @@ Julia aims to create an unprecedented combination of ease-of-use, power, and eff
 language. In addition to the above, some advantages of Julia over comparable systems include:
 
   * Free and open source ([MIT licensed](https://github.com/JuliaLang/julia/blob/master/LICENSE.md))
+  * Reliable built-in package manager that automatically installs compatible dependencies
+  * Talented, active, and welcoming community of developers
```

Comment:
Agree with @KristofferC. I would add that it is also difficult to quantify, so probably best let the links provided at the top of this file speak for the quality of the community.